services/shuffle/neff/mod.go

```diff
@@ -192,7 +189,7 @@ func (a *Actor) waitAndCheckShuffling(electionID string) error {
 		}
 
 		dela.Logger.Info().Msgf("waiting a while before checking election: %d", i)
-		time.Sleep(waitT)
+		time.Sleep(time.Second * time.Duration(rosterLen/2+1))
```

Comment:
Only for my understanding, why should the sleep be proportional to the roster size ?
Isn't the processing time correlated with the amount of votes that need to be shuffled ?

Reply:
The shuffling computation is one element, the other one is the consensus time and complexity, which grows with the number of servers.